enhancements/olm/olmv1-webhook-support.md

```diff
@@ -0,0 +1,380 @@
+2. The `service.beta.openshift.io/inject-cabundle: true` is present on all generated `ValidatingWebhookConfiguration`, `MutatingWebhookConfiguration`, and CustomResourceDefinitions that are configured for a webhook conversion strategy
+3. Updating webhook service backing deployment pod template specs to mount the tls configuration from the Openshift-ServiceCA generated `Secret`.
+
+Ensure parity with OLMv0 behavior by:
+1. Generating the `ValidatingWebhookConfiguration`, `MutatingWebhookConfiguration`, and `Service` resource with in the same way
+
+
+### Workflow Description
+
+There should be not workflow differences. User can install/upgrade bundles containing webhooks in the exact same way
+as they would for bundles that don't. This enhancement presents a broadening of the content that is consumable by users
+with no changes in user experience.
+
+### API Extensions
+
+N/A
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+OLMv1 does not yet support Hypershift. Given that the bundle installation happens on the user cluster, I don't believe
+there should be any specific Hypershift considerations. I.e. no resources will be installed in the central cluster.
+
+#### Standalone Clusters
+
+This change only affects the types of workloads that the customer will run. It should not have any major effect on 
+OLMv1 itself.
+
+#### Single-node Deployments or MicroShift
+
+This change only affects the types of workloads that the customer will run. It should not have any major effect on
+OLMv1 itself.
+
+### Implementation Details/Notes/Constraints
+
+The ClusterServiceVersion contained in the registry+v1 bundle to be installed is transpiled into the final set of
+manifests that will be applied to the cluster (i.e. CRDs, Deployments, Services, etc.). The ClusterServiceVersion
+can contain definitions for Validating, Mutating, and Conversion Webhooks, which form the basis for:
+- MutatingWebhookConfiguration resource(s)
+- ValidatingWebhookConfiguration resource(s)
+- Service resource(s) that back the webhooks
+- .spec.Conversion for any CRD resource with a conversion webhook
+
+The relevant generated resources, i.e. CRDs, Services, Mutating-/ValidatingWebhookConfigurations,
+will carry the appropriate Openshift-ServiceCA annotations for the generation and management TLS certificates for the
+webhook service(s). In addition, any Deployment resources that host the business logic for the webhook(s) will be 
+configured with the correct volumes and volume mounts to consume the generated TLS certificate.
+
+### Risks and Mitigations
+
+Because we are transpiling a ClusterServiceVersion into a set of coherent resources, there is a security implication that
+well crafted bundles _could_ potentially be malicious in nature and could be used to subvert to host cluster. These
+risks are mitigated by:
+- Allowing cluster admins to configure the scope of RBAC they are comfortable with for a particular bundle
```

Comment:
Is this the CE.spec.serviceAccount feature that we are planning to deprecate? If so, we should probably remove this line or update it to reflect our current thinking of a future approval flow.